tests/virt_autotest/set_config_as_glue.pm

```diff
@@ -31,7 +31,11 @@ sub fufill_guests_in_setting {
     my $get_vm_hostnames   = "virsh list  --all | grep sles | awk \'{print \$2}\'";
```

Comment:
Why don't you use some base function with an array containing names, addressess and other variables for every guest as we do in `lib/xen.pm`?

Reply:
We have different vm names and so mac addresses. Although we reuse variables and test functions from xen.pm, we still need to override it to load our vm info.

Reply:
OK, you can make this one as resolved.

Reply:
OK, now I understand. Please make this thread resolved.